settings.py

```diff
@@ -2329,3 +2329,10 @@ def _determine_couch_databases(couch_databases):
     ]
 
 PACKAGE_MONITOR_REQUIREMENTS_FILE = os.path.join(FILEPATH, 'requirements', 'requirements.txt')
+
+IS_LOCATION_CTE_ENABLED = UNIT_TESTING or SERVER_ENVIRONMENT in [
+    'localdev',
+    'changeme',  # default value in localsettings.example.py
+    'staging',
+    'softlayer',
```

Comment:
We can add more envs here after verifying that the timing looks good in datadog. I did this because there is now an extra CTE query executed for every locations ancestors/descendants MPTT query, and I didn't want to add that extra load to ICDS before verifying that the timings look good elsewhere.